tests/vmcheck/test-rojig-client.sh

```diff
@@ -30,6 +30,12 @@ if test "${osid}" != 'ID=fedora'; then
     exit 0
 fi
 
+$(vm_rpmostree --version) > version.txt
```

Comment:
Note this will still allocate a whole VM. We should probably enhance `filter_tests()` in a follow-up to just skip over tests with `rojig` in the name if it's compiled out.